aspnetcore/mvc/controllers/areas.md

```diff
@@ -185,13 +185,12 @@ Browsing to *http://<yourApp>/products*, the `Index` action method of the `HomeC
 
 ## Publishing Areas
 
-To publish all views of the areas folder, in the `project.json` file include an entry in the `publishOptions`'s `include` node like below:
+To publish all views of the areas folder, in the *.csproj* file include `Areas\**\*.cshtml` n the following markup:
 
-```text
-"publishOptions": {
-     "include": [
-       "Areas/**/*.cshtml",
-       ....
-       ....
-     ]
+```xml
+  <ItemGroup>
+    <Content Update="wwwroot\**\*;**\*.cshtml;Areas\**\*.cshtml;">
```

Comment:
I may be wrong, but I don't think this is required. As long as a user has `<Project Sdk="Microsoft.NET.Sdk.Web">`, all \*.cshtml and wwwroot/\** files are published to output.

cc @vijayrkn can you confirm?